### DISCUSSION TITLE
How to merge two row or column cells in table

### DISCUSSION THREAD

**Discussion by aksigkvgithub** [typst/typst#1944] [Q&A] [answered]
Many a times, we need two columns under one row header. And sometimes we need two rows into one header. Sometimes we simply need any number of cells to be merged and so on.

I am new to typst. I did some raw formatting in typst.

Any help is solicited.

Thanks and regards.

Dr. A. K. Singh

**Comment by LaurenzV** [comment, marked as answer]
You can try using the [tablex](https://github.com/typst/packages/tree/main/packages/preview/tablex/0.0.4) package.
